EAN_chksum.py

```diff
@@ -0,0 +1,35 @@
+# EAN8/EAN13/SSCC/GTIN Code Verifier & Checksum Generator
```

Comment:
Why I see the change towards this file in both client&server? 
Consider moving it to a seperate folder if this is an util tool for debugging/developing purpose.